demisto_sdk/commands/unify/yml_unifier.py

```diff
@@ -89,8 +89,11 @@ def __init__(self, input: str, dir_name=INTEGRATIONS_DIR, output: str = '',
         self.ryaml.preserve_quotes = True
         self.ryaml.width = 50000  # make sure long lines will not break (relevant for code section)
         if self.yml_path:
-            with io.open(self.yml_path, 'r', encoding='utf8') as yml_file:
-                self.yml_data = self.ryaml.load(yml_file)
+            if yml_modified_data:
```

Comment:
replace the order of the conditions:
if modified, use the given value.
otherwise, check the path